bundles/catalogue/metadataflyout/view/MetadataPanel.js

```diff
@@ -176,7 +176,7 @@ Oskari.clazz.define('Oskari.catalogue.bundle.metadataflyout.view.MetadataPanel',
                     '        </ul>' +
                     '    <% } %>' +
 
-                    '    <% if (identification.citation.date.date.length) { %>' +
+                    '    <% if (identification.citation.date.date != null && identification.citation.date.date.length) { %>' +
```

Comment:
If we can assume the type of `identification.citation.date.date` is always string or null/undefined, we could just use it directly as the condition in `if`:
```
<% if (identification.citation.date.date) { %>'
```
This will take care of both the null check and possible empty string, because an empty string is evaluated as "falsy".

Reply:
Yep, that is better.